### DISCUSSION TITLE
Shift+Enter same as CTRL-F?

### DISCUSSION THREAD

**Discussion by citizenmatt** [macvim-dev/macvim#1256] [Q&A] [answered]
Hi all. I've noticed that `<S-CR>` works the same as `<C-F>` in GUI MacVim. But in console Vim, `<S-CR>` acts the same as `<CR>`. Anyone know what's going on? I don't have any mappings - this was checked with `mvim -u NONE` and `vim -u NONE`.

**Comment by ychin** [comment, marked as answer]
This is the correct but confusing behavior. In Vim, `<CR>` acts just like `<Down>` in normal mode, which just moves cursor down by one line. If you do `:help <S-Down>`, it does the same thing as `<PageDown>` and `CTRL-F`, which also means `<S-CR>` does the same thing and move down half a page.

`<S-CR>` doesn't work in a lot of terminals though as this is not a mapping you can use. Thererfore it's just using the regular `<CR>`even when you press shift.

You can always remap `<S-CR>` to something else if there is  aparticualr need for it.

**Reply by citizenmatt** [reply]
Forgot to reply to this 😊 Thanks for the explanation!
